espnet/nets/pytorch_backend/e2e_tts_transformer.py

```diff
@@ -623,13 +623,15 @@ def inference(self, x, inference_args, *args, **kwargs):
 
         return outs, probs
 
-    def calculate_all_attentions(self, xs, ilens, ys, olens, *args, **kwargs):
+    def calculate_all_attentions(self, xs, ilens, ys, olens, skip_output=False, keep_tensor=False, *args, **kwargs):
```

Comment:
Noticed that args and kwargs are not used in the method. Can we remove them?

Reply:
This is needed to deal with different arguments between taco2 and transformer.
Strictly, not needed for transformer but needed for taco2.
But I want to use the same format of arguments.